Tooltips/Shaders/BlurDynamic.shader

```diff
@@ -0,0 +1,236 @@
+﻿Shader "EditorVR/Blur/Blur Dynamic"
```

Comment:
Getting the following error:

> Output value 'vert' is not completely initialized

on this and the other blur shader. But the when I reimport, it goes away.

Reply:
Odd, I'll look into that

Reply:
Pushed fix for fully initialized vertex program outputs